app/static/js/load_weeks.js

```diff
@@ -0,0 +1,43 @@
+document.onload = function () {
+    const container = document.querySelector('.calender-grid');
+}
+
+const anticipating = 1;
+const last = null;
+
+function open_daily(new_days) {
+    let all_days = new_days.getElementsByClassName('day'), i;
```

Comment:
Remove `, i`

Reply:
entered to the for loop it self as: (let i = 0; i < all_days.length; ++i)